client_cli/src/main.rs

```diff
@@ -159,21 +161,20 @@ fn main() -> Result<()> {
 /// Fails if submitting over network fails
 #[allow(clippy::shadow_unrelated)]
 pub fn submit(
-    instruction: impl Into<Instruction>,
+    instructions: Executable,
     cfg: &ClientConfiguration,
     metadata: UnlimitedMetadata,
 ) -> Result<()> {
-    let instruction = instruction.into();
     let iroha_client = Client::new(cfg)?;
     #[cfg(debug_assertions)]
     let err_msg = format!(
         "Failed to build transaction from instruction {:?}",
-        instruction
+        instructions
     );
     #[cfg(not(debug_assertions))]
     let err_msg = "Failed to build transaction.";
     let tx = iroha_client
-        .build_transaction(vec![instruction].into(), metadata)
+        .build_transaction(instructions, metadata)
```

Comment:
```suggestion
        .build_transaction(instructions, metadata)
```

This change is more conformant with the general design. Should we create an issue to submit multi-instruction transactions? Like a new subcommand that submits a JSON instruction set?

Reply:
Opened an issue
https://github.com/hyperledger/iroha/issues/2899